### DISCUSSION TITLE
apollo-react-10.0.0  JS version

### DISCUSSION THREAD

**Discussion by RamaKrishna-CEVA** [primefaces/community#595] [PrimeReact Templates] [answered]
Hi Team,

We downloaded the latest Apollo React template version 10.0.0, but in that zip file we found only the TypeScript version, can you please confirm JavaScript version is available or that only the TypeScript version only released?
If the JavaScript version is also available, please confirm the download location.

Thank you,
Ramakrishna Jetti

**Comment by buraksaglam089** [comment, marked as answer]
Hello,
We are only providing TypeScript versions since the decision for TypeScript was made in the community vote.
Regards.

**Comment by mertsincan** [comment]
Yes, we can also expect upvotes from other users about this. We will be happy to implement this when there are enough upvotes. Thanks a lot for your understanding!
